### ISSUE TITLE
Pokémon go version is updated

### ISSUE THREAD

**Issue by aquibzafar** [PokeBotPub/PokeBot-Android#242]
Does this mean we shouldn't use Pokebot anymore because of the whole updated version of Pokémon go ?

**Comment by ArthurPetrilli** [comment]
Hmmm, I read that the new update is goon be 0.49.1.  I am still using the bot on my 2nd acct.  I really wish i could use the bot on my main acct again!!!!  I just hatched a 10km egg.  I just got a Hitmonchan at 75% IV  What in the world is happening.  I have never seen eggs hatch with IV's less than 88%  I did all that F****ing walking and i get crappy IV's.  With the bot....i would just blow that off....But i did the manual work and i F***** starting to hate this game!!!!!!  It feels like i can't rely on eggs to get near perfect pokemon!!!!
